packages/web-runtime/src/components/Avatar.vue

```diff
@@ -88,20 +87,16 @@ export default {
         return
       }
       const accessToken = this.$store.getters['runtime/auth/accessToken']
-      const headers = new Headers()
       const instance = this.configuration.server || window.location.origin
       const url = instance + 'remote.php/dav/avatars/' + userid + '/128.png'
-      headers.append('Authorization', 'Bearer ' + accessToken)
-      headers.append('X-Requested-With', 'XMLHttpRequest')
-      headers.append('X-Request-ID', uuidV4())
-      fetch(url, { headers })
+      this.$clientService
+        .httpAuthenticated(accessToken)
+        .get(url, { responseType: 'blob' })
         .then((response) => {
-          if (response.ok) {
-            return response.blob()
-          }
-          if (response.status !== 404) {
-            throw new Error(`Unexpected status code ${response.status}`)
+          if (response.status === 200) {
+            return response.data
           }
+          throw new Error(`Unexpected status code ${response.status}`)
```

Comment:
The old code explicitly allowed `404` as soft failure. Because if a user doesn't have an avatar we'll legitimately have a 404. Any reason to change that?

Reply:
ah, no - we just had no response.ok to check for in the AxiosResponse.

We can check of status === 200 instead of checking for `ok`, good catch

Reply:
What I meant is that your new code throws the `Unexpected status code` error on a 404, which leads to the error being printed to the console further down in the `catch` block. We don't want that kind of noise. 😅  A 404 on an avatar request is just normal (not every user has an avatar), so both `200` and `404` are expected status codes.

Reply:
Understood, I just explained how we got there and how we can get back to the old behavior (which I'm in favor of)